test/test_region_of_interest.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+import numpy as np
+
+
+def test_region_of_interest_depth(file_camera):
+    import zivid
+
+    z_min = 600.0
+    z_max = 625.0
+
+    # Make sure default capture has points outside the range we want
+    settings = zivid.Settings(acquisitions=[zivid.Settings.Acquisition()])
```

Comment:
Good question. It works because of the clever fixture system we have here. This test depends on the `"file_camera"` fixture, which depends on the `"application"` fixture. The fixtures keep these things alive with a yield statement:

```
@pytest.fixture(name="application", scope="module")
def application_fixture():
    with zivid.Application() as app:
        yield app
```